cmd/mattermost/commands/export.go

```diff
@@ -180,7 +180,7 @@ func bulkExportCmdF(command *cobra.Command, args []string) error {
 	}
 	defer fileWriter.Close()
 
-	if err := a.BulkExport(fileWriter); err != nil {
+	if err := a.BulkExport(fileWriter, args[0]); err != nil {
```

Comment:
what is args[0] ?

Reply:
args[0] points to the file name (or file path) specified through the export bulk command as,

`export bulk demo.json --all-teams`

We are considering this argument to create a folder named `exported_emoji` in the same location of the file to export emojis from `data/emoji` to `exported_emoji`.

Reply:
It will still consider `demo.json` as Cobra differentiates between arguments and flags. 

I've tested this before replying here.